src/core/energy_storage/electric_storage.jl

```diff
@@ -148,10 +148,13 @@ end
     can_grid_charge::Bool = off_grid_flag ? false : true
     installed_cost_per_kw::Real = 910.0
     installed_cost_per_kwh::Real = 455.0
+    installed_cost_constant::Real = 0.0
     replace_cost_per_kw::Real = 715.0
     replace_cost_per_kwh::Real = 318.0
+    replace_cost_constant::Real = 0.0
     inverter_replacement_year::Int = 10
     battery_replacement_year::Int = 10
+    cost_constant_replacement_year::Int = 10
```

Comment:
@toddleif Can you explain the reasoning for making `cost_constant_replacement_year` a separate input, rather than just assuming the cost constant is associated with either the inverter/BOS or the battery packs?